schema/herocard-response-schema.json

```diff
@@ -74,7 +74,7 @@
         },
         "items": {
           "type": "array",
-          "fieldItems": {"$ref": "#/definitions/cardBodyFieldItem"},
+          "items": {"$ref": "#/definitions/cardBodyFieldItem"},
```

Comment:
As per the JSON schema convention, it should be "items" instead of "fieldItems".